cdt_identity/views.py

```diff
@@ -140,16 +136,15 @@ def logout(request: HttpRequest):
         # this does not look like an oauth_client, it's an error redirect
         return oauth_client_result
 
-    # overwrite the session token, the user is signed out of the app
-    token = session.oidc_token
-    session.clear_oidc_token()
+    post_logout = Routes.route_post_logout
+    if session.claims_request and session.claims_request.redirect_post_logout:
+        post_logout = session.claims_request.redirect_post_logout
 
-    route = reverse(Routes.route_post_logout)
-    redirect_uri = redirects.generate_redirect_uri(request, route)
+    post_logout_route = reverse(post_logout)
+    post_logout_uri = redirects.generate_redirect_uri(request, post_logout_route)
 
-    logger.debug(f"OAuth end_session_endpoint with redirect_uri: {redirect_uri}")
+    logger.debug(f"end_session_endpoint with redirect_uri: {post_logout_uri}")
 
     # send the user through the end_session_endpoint, redirecting back to
-    # the post_logout route
-    return redirects.deauthorize_redirect(request, oauth_client, token, redirect_uri)
-    return redirects.deauthorize_redirect(request, oauth_client, token, redirect_uri)
+    # the post_logout URI
+    return redirects.deauthorize_redirect(request, oauth_client, post_logout_uri)
```

Comment:
I don't see a change in this PR to the parameters of [`redirects.deauthorize_redirect`](https://github.com/Office-of-Digital-Services/django-cdt-identity/blob/main/cdt_identity/redirects.py#L8)... would `post_logout_uri` end up being passed as the argument for `token` ?

https://github.com/Office-of-Digital-Services/django-cdt-identity/blob/8a30a026a1d4efdc65e2ee26c0e7749eb17cda0a/cdt_identity/redirects.py#L8-L22

Also wondering if we want to go ahead and update the line that creates the `params` dict to not include `id_token_hint` per #14, or if that should be saved for a separate PR:

https://github.com/Office-of-Digital-Services/django-cdt-identity/blob/8a30a026a1d4efdc65e2ee26c0e7749eb17cda0a/cdt_identity/redirects.py#L18

Reply:
[`f137c12`](https://github.com/Office-of-Digital-Services/django-cdt-identity/pull/9/commits/f137c1255c4e831546c8d9738d1746d8bb7a58c0)